agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/init/NettyVersions.java

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.microsoft.applicationinsights.agent.internal.init;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.nio.charset.StandardCharsets;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+
+class NettyVersions {
+
+  // Example: **19** **Group:** `io.netty` **Name:** `netty-codec-dns` **Version:** `4.1.79.Final`
+  private static final Pattern DEPENDENCY_COORDINATE_PATTERN =
+      Pattern.compile(".*`(.*)`.*`(.*)`.*`(.*)`");
+
+  private NettyVersions() {}
+
+  static String extract() {
+    String moreLicenseResource = "/META-INF/licenses/more-licenses.md";
+    InputStream moreLicenseAsStream = NettyVersions.class.getResourceAsStream(moreLicenseResource);
+    if (moreLicenseAsStream == null) {
+      return moreLicenseResource + " notFound";
```

Comment:
when customer is not using netty, i think it's better not to log it at all. 'not found' for something that they don't use is a bit confusing.

Reply:
`META-INF/licenses/more-licenses.md` file is supposed to always exist in Application Insights jar. It contains the versions of some libraries used by Application Insights.

Reply:
what about `NettyVersions.class.getResourceAsStream(moreLicenseResource)`?